mitmproxy/io/compat.py

```diff
@@ -142,7 +159,7 @@ def _convert_dict_keys(o: Any) -> Any:
 def _convert_dict_vals(o: dict, values_to_convert: dict) -> dict:
     for k, v in values_to_convert.items():
         if not o or k not in o:
-            continue
+            continue  # pragma: no cover
```

Comment:
This line is reached in tests but marked as not covered due python's optimization ([related](https://bitbucket.org/ned/coveragepy/issues/198/continue-marked-as-not-covered#comment-4052311))